nest/02.nest.md

```diff
@@ -0,0 +1,99 @@
+# 2 nest
+In this chapter, we will detail how to use the command `nest`.
+As explained before, this command is mainly designed for ease of use.
+However, anyone can use it.
+
+## 2.1 nest commands
+### 2.1.1 Updating the list of available packages with `nest pull`
+Nest uses a local cache to store the list of available packages you can install.
+When you're installing a package, Nest will search for it in this cache.
+The command to update the cache containing the list of available packages, is `nest pull`.
+
+Its output looks like this:
+
+```
+$ nest pull
+1 pending transaction:
+
+    pull stable
+
+Would you like to apply this transaction? [Yes/no] 
+Pulling stable...
+Successfully pulled 1 repository
+```
+
+> Attention
+>
+> Nest won't be able to install packages that are not in your cache, so it's really important to keep your cache up-to-date as often as possible.
+> If a new package or a new version of a package comes out after the last time you updated the cache, Nest won't be able to install it.
+> It's recommended to run `nest pull` at least before installing and before upgrading a package.
+
+#### Common issues
+* `error: unable to pull repository stable: no working mirror found`
+
+[//]: # (TODO: add link to the section 3.1.3)
+Basically, this error means that `nest` couldn't pull the repository `stable`.
+The most common reason is that there's an issue with your Internet connection.
+It might also be that all the mirrors in the repositories configuration file are down, so you need to add working mirrors to the list.
+You can have more information about this file and mirrors in [the section 3.1.3]().
+
+### 2.1.2 Installing and uninstalling a package
+#### Installing a package with `nest install`
+[//]: # (TODO: add link to the section 1.4)
+`nest install` is the command to install a package.
+To do so, run `nest install <package>` in a terminal.
+Don't forget to use the syntax explained in [section 1.4]() for the package name!
+
+`nest` will print an error message if more than one package matches the syntax you wrote.
+In that case, you should first refine your package name.
+To help you with that, you can go on [the official Raven-OS website](https://stable.raven-os.org), which will show you all the packages in the `stable` repository, with their category and all  versions released.
+
+For example, the output of `nest install stable::sys-bin/ncurses6-bin` is as followed:
```

Comment:
Why doesn't the example match ? This is can be really confusing.
Also, maybe just a simple "Example:" may be enough.